src/app/pages/data-protection/cloudsync/cloudsync-list/cloudsync-list.component.ts

```diff
@@ -98,10 +98,11 @@ export class CloudsyncListComponent implements EntityTableConfig<CloudSyncTaskUi
   resourceTransformIncomingRestData(data: CloudSyncTask[]): CloudSyncTaskUi[] {
     return data.map((task) => {
       const transformed = { ...task } as CloudSyncTaskUi;
+      const formattedCronSchedule = `${task.schedule.minute} ${task.schedule.hour} ${task.schedule.dom} ${task.schedule.month} ${task.schedule.dow}`;
       transformed.credential = task.credentials.name;
-      transformed.cron_schedule = `${task.schedule.minute} ${task.schedule.hour} ${task.schedule.dom} ${task.schedule.month} ${task.schedule.dow}`;
-      transformed.frequency = this.taskService.getTaskCronDescription(transformed.cron_schedule);
-      transformed.next_run = this.taskService.getTaskNextRun(transformed.cron_schedule);
+      transformed.cron_schedule = task.enabled ? formattedCronSchedule : T('Disabled');
+      transformed.frequency = this.taskService.getTaskCronDescription(formattedCronSchedule);
```

Comment:
Should this be conditional as well? Based on whether task is enabled/disabled? Just curious. Not a recommendation.

Reply:
It was not requested in the ticket but just asked for schedule and next_run.